examples/tutorials/01_basics/07_backpack_lights.py

```diff
@@ -35,6 +35,9 @@ def cozmo_program(robot: cozmo.robot.Robot):
     # set all of Cozmo's backpack lights to blue, and wait for 2 seconds
     robot.set_all_backpack_lights(cozmo.lights.blue_light)
     time.sleep(2)
+    # set just Cozmo's center backpack lights to white, and wait for 2 seconds
+    robot.set_center_backpack_lights(cozmo.lights.white_light)
```

Comment:
off topic but i wish i'd named all those light related constants in caps to be consistent with other constants.. brain fart on my part.